### DISCUSSION TITLE
Best practice to hide DOC TOC on breakpoints

### DISCUSSION THREAD

**Discussion by shipswake** [facebook/docusaurus#10478] [General] [answered]
When in smaller screens what is the best method to hide or minimize the doc toc?

For example: when viewing in an iPhone I do not want to display the doc toc full height

**Comment by slorber** [comment]
What does it mean to hide or minimize the TOC? It's already converted to a dropdown for small screens <= 996px

![CleanShot 2024-09-05 at 17 27 58](https://github.com/user-attachments/assets/af11c452-0abf-47d0-bf02-77862cff9eac)

Please be precise. What is the exact width, what do you see, and what do you want to see instead?

**Reply by shipswake** [reply]
This is what I see 
The dropdown fills the page

![image](https://github.com/user-attachments/assets/c6f2f1ff-59ce-4ce5-9e39-553274bca91b)

**Reply by slorber** [reply, marked as answer]
This is not what we provide out of the box, you must have broken something with custom code or CSS.
